### DISCUSSION TITLE
best funding method

### DISCUSSION THREAD

**Discussion by bjlockie** [FossifyOrg/General-Discussion#404] [General]
I set up a Patreon and then read LiberPay doesn't take a cut but it says "Donations to naveensingh are processed through PayPal." so maybe I should use Paypal directly.
Plus there will be a currency conversion involved so what is the best way to contribute?

**Comment by naveensingh** [comment]
@bjlockie 

Until my Stripe account is approved for Liberapay, I think **Github Sponsors** is the only option with [little to no fees](https://docs.github.com/en/sponsors/getting-started-with-github-sponsors/about-github-sponsors#about-github-sponsors): 

> GitHub Sponsors does not charge any fees for sponsorships from personal accounts, so 100% of these sponsorships go to the sponsored developer or organization. GitHub Sponsors charges a fee of up to 6% for sponsorships from organization accounts.

So unless you are an organization, there shouldn't be any fee.

Thanks!
